src/familiar/index.ts

```diff
@@ -7,6 +7,8 @@ import { canOpenRedPresent, pocketProfessorLectures, timeToMeatify } from "./lib
 export { canOpenRedPresent, timeToMeatify, pocketProfessorLectures, freeFightFamiliar };
 
 export const chooseFamiliar = () =>
-  $familiars`Reagnimated Gnome, Temporal Riftlet`.find((f) => have(f)) ?? freeFightFamiliar();
+  canInteract()
```

Comment:
this is a clever way to support loopin' good call on can_interact rather than a manual heck for gyou

Reply:
hell yeah